twitter_favorite_block.py

```diff
@@ -10,32 +10,32 @@
 class TwitterFavorite(TwitterRestBase):
 
     version = VersionProperty("1.0.0")
-    id = Property(default='{{$id}}', title='Tweet ID')
+    tweet_id = Property(default='{{$tweet_id}}', title='Tweet ID')
```

Comment:
I think the old default value was  better. Renaming the block's `id` variable to `tweet_id` in this case was the right thing to do, but the signals being passed to this block are presumably tweets, and from the [twitter docs here](https://developer.twitter.com/en/docs/tweets/data-dictionary/overview/tweet-object) it looks like `id` is the field name that would be used. I appreciate the disambiguation, but this reference to `$id` is safe.